### PR TITLE
Realtime fix

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.1.4
- Fixed: Set correct is realtime flag for recordings

### CODE REVIEW

File: src/client.cpp [hunk 1]
```diff
@@ -52,6 +52,7 @@ std::string             g_szUserPath   = "";
 std::string             g_szClientPath = "";
 CHelper_libXBMC_addon  *XBMC           = NULL;
 CHelper_libXBMC_pvr    *PVR            = NULL;
+bool                   b_recordingPlayback = false;
```
Comment (ksooo) [inline src/client.cpp]: strange naming. Maybe this should read g_b* to denote a global var?

Comment (phunkyfish) [reply src/client.cpp]: Sorry, should have been m_bRecordingPlayback.

It’s not actually global at all.

Comment (ksooo) [reply src/client.cpp]: I'm confused. AFAIR we use m_ only for class member vars. But you are right with this var not being a global var. It's visible throughout the compilation unit only. Shouldn't it be 'static', then?

Comment (phunkyfish) [reply src/client.cpp]: Hmmm. Or just `bRecordingPlayback`?

Looking at several addons they appear to use either an `m_` or omit it in client.cpp.

No addon I can see uses static in this file.

Comment (phunkyfish) [reply src/client.cpp]: I will go with `m_`. Because when we go to C++ this will be member variable.

**Pull request merged**